Buy/Utilities/Global.swift

```diff
@@ -39,7 +39,7 @@ class Global {
     //  MARK: - Versions -
     //
     static var frameworkVersion: String {
-        return self.frameworkBundle.object(forInfoDictionaryKey: "CFBundleShortVersionString") as! String
+        return "\(Storefront.Schema.version)\(SDK.version)"
```

Comment:
Should there be some sort of delimiter between these two values? Not sure if it matters for potential server-side reporting.

Reply:
Ah, yes. Good catch. There was supposed to be a `/` in there. Got lost in all the `\`.

Reply:
nitpick: should it be the other way around? `5.1.0/2020-07`?

Reply:
The rationale was that schema version is likely to stay the same _longer_ than SDK version. For example, a subsequent release might "fix bugs" and so the version becomes - `Mobile Buy SDK iOS/2020-07/5.1.1`. Ultimately, I don't have a strong preference here.

Reply:
Ah, I didn't see the `User-Agent` established format. My opinion would be for using the SemVer approach of `5.1.0+2020-07` as the framework version really is a composite of the two versions, but I am also not strongly held in that regard.